app/src/main/java/net/skyscanner/backpack/demo/compose/ProgressBarStory.kt

```diff
@@ -0,0 +1,122 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.demo.compose
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import net.skyscanner.backpack.compose.button.BpkButton
+import net.skyscanner.backpack.compose.progressbar.BpkProgressBar
+import net.skyscanner.backpack.compose.progressbar.BpkProgressBarSize
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.demo.components.ProgressBarComponent
+import net.skyscanner.backpack.demo.meta.ComposeStory
+import java.lang.Float.max
+import java.lang.Float.min
+
+@Composable
+@ProgressBarComponent
+@ComposeStory
+fun ProgressBarStory(modifier: Modifier = Modifier) {
+
+    Column(modifier = modifier.padding(BpkSpacing.Lg)) {
+
+        ProgressBarSample(
+            title = "Small",
```

Comment:
nit: can you use string resources please?

Reply:
Even for story? Why?

Reply:
we're still trying to follow general standards/best practices here